I applied this change and reinstalled amrex but I'm seeing errors finding FFTW in the test:

```
CMake Error at /root/spack/opt/spack/linux-x86_64_v3/cmake-3.31.6-ild7ovc5pqaxz7h6ecdmhpozpfvfb36k/share/cmake-3.31/Modules/FindPackageHandleStandardArgs.cmake:233 (message):
  Could NOT find PkgConfig (missing: PKG_CONFIG_EXECUTABLE)
Call Stack (most recent call first):
  /root/spack/opt/spack/linux-x86_64_v3/cmake-3.31.6-ild7ovc5pqaxz7h6ecdmhpozpfvfb36k/share/cmake-3.31/Modules/FindPackageHandleStandardArgs.cmake:603 (_FPHSA_FAILURE_MESSAGE)
  /root/spack/opt/spack/linux-x86_64_v3/cmake-3.31.6-ild7ovc5pqaxz7h6ecdmhpozpfvfb36k/share/cmake-3.31/Modules/FindPkgConfig.cmake:114 (find_package_handle_standard_args)
  /root/spack/opt/spack/linux-x86_64_v3/amrex-25.06-unfn6hnafsrq5bqffkdezyvtymv4hzs2/lib/cmake/AMReX/AMReXCMakeModules/FindAMReXFFTW.cmake:111 (find_package)
  /root/spack/opt/spack/linux-x86_64_v3/amrex-25.06-unfn6hnafsrq5bqffkdezyvtymv4hzs2/lib/cmake/AMReX/AMReXCMakeModules/FindAMReXFFTW.cmake:143 (fftw_find_precision)
  /root/spack/opt/spack/linux-x86_64_v3/cmake-3.31.6-ild7ovc5pqaxz7h6ecdmhpozpfvfb36k/share/cmake-3.31/Modules/CMakeFindDependencyMacro.cmake:76 (find_package)
  /root/spack/opt/spack/linux-x86_64_v3/amrex-25.06-unfn6hnafsrq5bqffkdezyvtymv4hzs2/lib/cmake/AMReX/AMReXConfig.cmake:237 (find_dependency)
  CMakeLists.txt:39 (find_package)

```

--- a/repos/spack_repo/builtin/packages/amrex/package.py
+++ b/repos/spack_repo/builtin/packages/amrex/package.py
@@ -4,7 +4,7 @@
 
 import os
 
-from spack_repo.builtin.build_systems.cmake import CMakePackage
+from spack_repo.builtin.build_systems.cmake import CMakePackage, get_cmake_prefix_path
 from spack_repo.builtin.build_systems.cuda import CudaPackage
 from spack_repo.builtin.build_systems.rocm import ROCmPackage
 
@@ -433,14 +433,10 @@ class Amrex(CMakePackage, CudaPackage, ROCmPackage):
 
         if self.spec.satisfies("+cuda"):
             args.append("-DCMAKE_CUDA_COMPILER=" + join_path(self.spec["cuda"].prefix.bin, "nvcc"))
+            args.append("-DCMAKE_INSTALL_RPATH=" + self.spec["cuda"].prefix.lib64)
+            args.append("-DCMAKE_BUILD_WITH_INSTALL_RPATH=ON")
 
-        prefix_paths = []
-        for dep in ["fftw", "hdf5", "hypre", "petsc", "sundials"]:
-            if dep in self.spec:
-                prefix_paths.append(self.spec[dep].prefix)
-
-        if prefix_paths:
-            args.append("-DCMAKE_PREFIX_PATH=" + ";".join(prefix_paths))
+        args.append("-DCMAKE_PREFIX_PATH=" + ";".join(get_cmake_prefix_path(self)))
 
         args.extend(self.cmake_args())
         cmake = which(self.spec["cmake"].prefix.bin.cmake)

